source/blog/articles/2019-07-onward-and-upward.html.markdown

```diff
@@ -0,0 +1,80 @@
+
+But soon after leaving the train and starting the hike another stray dog took a liking to us and joined in on the hike! The dog followed us all the way to the top (which was an exhausting 5.75 miles and a height of 2731 feet). The dog didn't seem the least bit phased by the climb and I must say strapping my pack to the dogs back was a thought that crossed my mind.
+
+Each of us had a pack weighing from 20 to 30 some pounds so the hike was no small challenge!
+
+On our way down we met up with a group of 11 teenage girls who had all but run out of water and they still had a good bit of hiking ahead of them so we were able to share some of our water with them which they were very thankful for.
+
+After coming off the top we set up camp and started a fire to make us some super on, then afterwards we sat around the fire and had a good time of fellowship. By the time we got in our tents the sleeping bag felt like a bed of feathers.
+
+The following morning we packed up our tents and started the agonizing descent off the mountain. Ben, Nathan, Corny, and Ralph took their time coming off the mountain (which was a smart idea) and made it in 2.5 hours,
+
+Kade and I on the other hand had a little competition going on and on the final stretch it turned into a race (I paid for it with a blister on my foot). Kade won the race and our finish time was 1 hour 43 minutes!
+
+On the following day (Friday 28th) Ben brought a devotion on being all in for God. Afterwards we did book reports. Each of us had read a book and shared what we had learned. That was very interesting.
+
+In the afternoon all of us went down to Center and did Bible surveys. After each survey we would gave participants a copy of Good and Evil. After we finished the surveys we set up the chalkboard. Then Joshua and Abigail sang their duet followed by Ralph, Ben, and Kade presenting the Gospel! The crowd was attentive and the messages were great!
+
+In the evening Joshua and Kelsie had everyone over for dinner as Ben was leaving for the States in the morning.
+
+The next morning Nathan had devotion with the topic of being a good steward with our finances and with the time and talents God has given us to serve Him.  
+
+That afternoon we went to Zhovkva and did more Bible surveys. Then we came back to Lviv in hopes of doing street preaching, but we weren't able to obtain a permit on that day so we ended up doing more surveys. This turned out to be pretty amazing because as Joshua and Ralph were doing surveys a man called over to them saying he would like to take the survey! He was a Catholic who for the past 5 months has been reading the Bible and had some questions for them! So Joshua talked with him for a while and gave him a Good and Evil book and a Bible First correspondence course. It's amazing to see the hand of God at work!
```

Comment:
Let's add an apostrophe in "Lviv" at the start of the paragraph.

Reply:
Did a find/replace to make sure this was consistent throughout.